htdocs/accountancy/admin/account.php

```diff
@@ -314,7 +314,7 @@
 }
 
 if (is_array($search_categories) && count($search_categories) > 0) {
-	$sql .= ' AND ac.rowid in ('.implode(',', $search_categories).')';
+	$sql .= ' AND ac.rowid IN ('.$db->escape(implode(',', $search_categories)).')';
```

Comment:
The quote to use in PHP to forge the string must be " instead of '.
$sql .= " ...
Inside the IN you must use  $db->sanitize and not $db->escape

Reply:
Done